cogs/wordfilter/wordfilter.py

```diff
@@ -403,14 +402,13 @@ def __init__(self, bot: Red):
                 if checkMsg.startswith(prefix+cmd):
                     blacklistedCmd = True
 
-        for word in filteredWords:
-            try:
-                filteredMsg = _filterWord(word, filteredMsg)
-            except Exception as error: # pylint: disable=broad-except
-                self.logger.error("Exception!")
-                self.logger.error(error)
-                self.logger.info("Word: %s", word)
-                self.logger.info("Filtered message: %s", filteredMsg)
+        try:
+            filteredMsg = _filterWord(filteredWords, filteredMsg)
+        except Exception as error: # pylint: disable=broad-except
+            self.logger.error("Exception!")
+            self.logger.error(error)
+            self.logger.info("Word: %s", word)

```

Comment:
This `word` variable no longer exists, since you removed the loop.

Reply:
That's from before I started editing it. Both `*` and `{}` are repetition for get rid of one.

Reply:
It looks like `re.sub` can only except if the regex is invalid. So maybe we can just remove that exception?

Reply:
Sure, let's take out this line, and change this except block to be more specific and catch `re.error` instead.